app/views/publishers/home.html.slim

```diff
@@ -210,32 +210,42 @@ script type="text/html" id="confirm_default_currency_modal_wrapper"
                       = publisher_channel_balance(current_publisher, channel.details.channel_identifier, "BAT")
                     span.bat-channel--currency= " BAT"
                     .bat-channel--period
-                      = t(".channel_balance_period")
-            - elsif channel.verification_started? || channel.verification_failed?
+                      = t(".channel_balance_period")            
+              .channel-summary
+                h3= channel.publication_title
+                .channel-details
+                  .added-date
+                    = t ".channel.added", date: channel.created_at.to_date.iso8601
+            - elsif channel.verification_failed?
               .channel-status.float-right
-                = channel_verification_details(channel)
-                = link_to(t(".channel.try_again"), channel_next_step_path(channel), class: "btn btn-primary")
+                .verification-failed
```

Comment:
IMO we should start breaking up the `home.html.slim` and merge as part of a `dashboard_partials` or `home_partials` folder just to make the folder not bloated.